### PR TITLE
Implement localizations mechanism, add Spanish localization as first #added

### PR DESCRIPTION
**Pull request by Kaspik**
## Changes:
- add fastlane automation for localizations
- add crowdin script for automated localizations
- localize Localizable.strings and xibs

## Closes following issues:
- Closes https://github.com/Sequel-Ace/Sequel-Ace/issues/499

## Tested:
- Processors:
  - [x] Intel
  - [ ] Apple Silicon
- macOS versions:
  - [ ] 10.12
  - [ ] 10.13
  - [ ] 10.14
  - [ ] 10.15
  - [x] 11.0
- Xcode version: 12.2

### CODE REVIEW

File: Scripts/translations-fetch.sh [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+for CURRENT_DIR in */ ; do
+
+  #This will hold the name that we want the language to be
+  TARGET_DIR=$CURRENT_DIR
+
+  DIRECTORY="${OUTPUT_DIR}${TARGET_DIR}"
+
+  if [ -d "$DIRECTORY" ]; then
+    echo "Updating langauge for code: $TARGET_DIR"
```
Comment (samuelbradshaw) [inline Scripts/translations-fetch.sh:81]: Typo (langauge -> language) :)

**Comment by pulsedrivervstibby** [comment]
Hello Kaspik,

i see in source code localizable.strings, can you add the others strings files (MainMenu.strings, Preferences.strings, AboutPanel.strings, etc...)

Thank you for all.

Regards.  PvD.

**Comment by Kaspik** [comment]
@pulsedrivervstibby We don't have `.xib`s localized yet (MainMenu is `.xib` file, which by localizing creates another xib file - not strings file), we need to migrate to storyboards unfortunately - so some parts of the UI can't be localized just yet... I'm trying to move forward as quickly as we can, but moving the UI to modern code will take some time unfortunately.

Also, please really don't use just the strings as I'll need to replace them in Crowdin anyways, so direct usage of Crowdin is preferred. :)

**Comment by pulsedrivervstibby** [comment]
Perfect, I begin with localizable.strings.

**Comment by Jason-Morcos** [comment]
> Perfect, I begin with localizable.strings.

Please use Crowdin for localization. I won't accept a PR directly to localizable.strings. 

Please fill out the following contact us form with your email address and we can get you added to Crowdin. https://moballo.com/contact-us/

**Pull request merged**